### PR TITLE
Add FTA Notes report and Cycle Time reports

### PR DESCRIPTION
**Pull request by FletcherS7**
Adds a CSV report with all the notes for teams added from the FTA Field Monitor.
Adds Match Cycle time reports for Practice, Qualifications, and Eliminations.

### CODE REVIEW

**Comment by FletcherS7** [comment]
Closes https://github.com/Team254/cheesy-arena/issues/126

File: field/arena.go [hunk 1]
```diff
@@ -855,6 +855,7 @@ func (arena *Arena) handlePlcOutput() {
 		// Turn off lights if all teams become ready.
 		if redAllianceReady && blueAllianceReady {
 			arena.Plc.SetFieldResetLight(false)
+			if arena.CurrentMatch.FieldReadyAt.IsZero() { arena.CurrentMatch.FieldReadyAt = time.Now()}
```
Comment (patfair) [inline field/arena.go:858]: Please run `go fmt ./...` in the repo root

File: web/web.go [hunk 2]
```diff
@@ -158,12 +158,14 @@ func (web *Web) newHandler() http.Handler {
 	router.HandleFunc("/reports/csv/schedule/{type}", web.scheduleCsvReportHandler).Methods("GET")
 	router.HandleFunc("/reports/csv/teams", web.teamsCsvReportHandler).Methods("GET")
 	router.HandleFunc("/reports/csv/wpa_keys", web.wpaKeysCsvReportHandler).Methods("GET")
+	router.HandleFunc("/reports/csv/fta", web.ftaCsvReportHandler).Methods("GET")
```
Comment (patfair) [inline web/web.go:161]: Please maintain alphabetical ordering of routes

File: web/web.go [hunk 3]
```diff
@@ -158,12 +158,14 @@ func (web *Web) newHandler() http.Handler {
...
 	router.HandleFunc("/reports/pdf/alliances", web.alliancesPdfReportHandler).Methods("GET")
 	router.HandleFunc("/reports/pdf/backups", web.backupsPdfReportHandler).Methods("GET")
 	router.HandleFunc("/reports/pdf/bracket", web.bracketPdfReportHandler).Methods("GET")
 	router.HandleFunc("/reports/pdf/coupons", web.couponsPdfReportHandler).Methods("GET")
 	router.HandleFunc("/reports/pdf/rankings", web.rankingsPdfReportHandler).Methods("GET")
 	router.HandleFunc("/reports/pdf/schedule/{type}", web.schedulePdfReportHandler).Methods("GET")
+	router.HandleFunc("/reports/pdf/cycle/{type}", web.cyclePdfReportHandler).Methods("GET")
```
Comment (patfair) [inline web/web.go:168]: Same

File: field/arena.go (hunk 1 above)
Comment (FletcherS7) [reply field/arena.go:858]:
```
main.go
bracket/bracket.go
bracket/bracket_test.go
bracket/double_elimination.go
bracket/double_elimination_test.go
bracket/matchup.go
bracket/matchup_test.go
bracket/single_elimination.go
bracket/single_elimination_test.go
bracket/test_helpers.go
field/arena.go
field/arena_notifiers.go
field/arena_test.go
field/display.go
field/display_test.go
field/driver_station_connection.go
field/driver_station_connection_test.go
field/event_status.go
field/event_status_test.go
field/realtime_score.go
field/scoring_panel_registry.go
field/scoring_panel_registry_test.go
field/team_match_log.go
field/test_helpers.go
game/foul.go
game/hub.go
game/hub_test.go
game/match_sounds.go
game/match_timing.go
game/ranking_fields.go
game/ranking_fields_test.go
game/rule.go
game/rule_test.go
game/score.go
game/score_summary.go
game/score_summary_test.go
game/score_test.go
game/test_helpers.go
model/alliance.go
model/alliance_test.go
model/award.go
model/award_test.go
model/database.go
model/database_test.go
model/event_settings.go
model/event_settings_test.go
model/lower_third.go
model/lower_third_test.go
model/match.go
model/match_result.go
model/match_result_test.go
model/match_test.go
model/ranking.go
model/ranking_test.go
model/schedule_block.go
model/schedule_block_test.go
model/sponsor_slide.go
model/sponsor_slide_test.go
model/table.go
model/table_test.go
model/team.go
model/team_test.go
model/test_helpers.go
model/user_session.go
model/user_session_test.go
network/access_point.go
network/access_point_test.go
network/switch.go
network/switch_test.go
partner/tba.go
partner/tba_test.go
plc/armorblock_string.go
plc/coil_string.go
plc/input_string.go
plc/plc.go
plc/plc_test.go
plc/register_string.go
tournament/awards.go
tournament/awards_test.go
tournament/qualification_rankings.go
tournament/qualification_rankings_test.go
tournament/schedule.go
tournament/schedule_test.go
tournament/test_helpers.go
web/alliance_selection.go
web/alliance_selection_test.go
web/alliance_station_display.go
web/alliance_station_display_test.go
web/announcer_display.go
web/announcer_display_test.go
web/api.go
web/api_test.go
web/audience_display.go
web/audience_display_test.go
web/bracket_display.go
web/bracket_display_test.go
web/display_utils.go
web/field_monitor_display.go
web/field_monitor_display_test.go
web/login.go
web/login_test.go
web/match_play.go
web/match_play_test.go
web/match_review.go
web/match_review_test.go
web/placeholder_display.go
web/placeholder_display_test.go
web/queueing_display.go
web/queueing_display_test.go
web/rankings_display.go
web/rankings_display_test.go
web/referee_panel.go
web/referee_panel_test.go
web/reports.go
web/reports_test.go
web/scoring_panel.go
web/scoring_panel_test.go
web/setup_awards.go
web/setup_awards_test.go
web/setup_displays.go
web/setup_displays_test.go
web/setup_field_testing.go
web/setup_field_testing_test.go
web/setup_lower_thirds.go
web/setup_lower_thirds_test.go
web/setup_schedule.go
web/setup_schedule_test.go
web/setup_settings.go
web/setup_settings_test.go
web/setup_sponsor_slides.go
web/setup_sponsor_slides_test.go
web/setup_teams.go
web/setup_teams_test.go
web/twitch_display.go
web/twitch_display_test.go
web/web.go
web/web_test.go
websocket/notifier.go
websocket/notifier_test.go
websocket/websocket.go
websocket/websocket_test.go
```

**Pull request merged**